CHANGELOG.md

```diff
@@ -14,6 +14,15 @@ way to update this template, but currently, we follow a pattern:
 
 ## Upcoming version 2021-XX-XX
 
+- [fix] Checkout process had bugs:
```

Comment:
Tiny thing, but this just tells that there were bugs, not that those bugs were fixed. :D

Maybe something like:

```suggestion
- [fix] Fix bugs in checkout process:
```

Reply:
It felt strange to write `[fix] Fix`